website/src/pages/tech-specs.md

```diff
@@ -0,0 +1,371 @@
+# Apache Hudi Storage Format Specification [DRAFT]
+
+
+
+This document is a specification for the Hudi Storage Format which transforms immutable cloud/file storage systems into transactional data lakes. 
+
+## Overview
+
+Hudi Storage Format enables the following features over very large collection of files/objects
```

Comment:
Table Format is more precise? storage format is a bit generic and i've noticed people sometimes inaccurately refer to it as file format.

Reply:
I dont want to call the spec as table format. Table format to me refers to catalogs, schema, object definition etc. Storage format is the spec for the storage layer. Let us think it over.